src/test/model-viewer-base-spec.ts

```diff
@@ -208,6 +208,63 @@ suite('ModelViewerElementBase', () => {
           expect(dataUrlMatcher.test(element.toDataURL())).to.be.true;
         });
       });
+
+      suite('toBlob', () => {
+        test('produces a blob', async () => {
+          const blob = await element.toBlob();
+          expect(blob).to.not.be.null;
+        });
+
+        test('can convert blob to object URL', async () => {
+          const blob = await element.toBlob();
+          const objectUrl = URL.createObjectURL(blob);
+          const objectUrlMatcher = /^blob\:https?:\//;
```

Comment:
FYI this is the format of the object URL that I get in IE11 when doing these steps manually: `blob:B0C5923D-DC2D-4666-9362-3AFDD6BADC30`

Reply:
Should we just match: `/^blob\:/` then?

Reply:
That's fine with me